docs/scos/dev/glue-api-guides/202200.0/managing-wishlists/managing-wishlist-items.md

```diff
@@ -0,0 +1,258 @@
+| configuration  | Array  | Default product configuration.  |
+| configuratorKey  | String  | Configurator type.  |
+| isComplete  | Boolean  | Shows if the configurable product configuration is complete:<div><ul><li>`true`—configuration complete.</li><li>`false`—configuration incomplete.</li></ul></div>  |
+| quantity  | Integer  | Quantity of the product that was added to the wishlist.  |
+| availableQuantity  | Integer  | Product quantity available in the store. |
+
+For attribute descriptions of product prices, see [Retrieving abstract product prices](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-products/abstract-products/retrieving-abstract-product-prices.html#abstract-product-prices-response-attributes).
+
+For attribute desriptions of concrete product availabillity, see [Retrieving concrete product availability](/docs/scos/dev/glue-api-guides/202108.0/managing-products/concrete-products/retrieving-concrete-product-availability.html#concrete-product-availability-response-attributes).
+
+## Delete a wishlist item
+
+To delete wishlist item, send the request:
+
+---
+`DELETE` **/wishlists/*{% raw %}{{{% endraw %}wishlist_id{% raw %}}}{% endraw %}*/wishlist-items/*{% raw %}{{{% endraw %}item_sku{% raw %}}}{% endraw %}***
+
+---
+
+| PATH PARAMETER | DESCRIPTION |
+| --- | --- |
+| ***{% raw %}{{{% endraw %}wishlist_id{% raw %}}}{% endraw %}*** | Unique identifier of the wishlist to delete an item from. [Create a wishlist](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-wishlists/managing-wishlists.html#create-a-wishlist) or [retrieve all wishlists](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-wishlists/managing-wishlists.html#retrieve-wishlists) to get it. |
+| ***{% raw %}{{{% endraw %}item_sku{% raw %}}}{% endraw %}*** | Unique identifier of the product to delete. |
+
+### Request
+
+| HEADER KEY | HEADER VALUE | REQUIRED | DESCRIPTION |
+| --- | --- | --- | --- |
+| Authorization | string | &check; | Alphanumeric string that authorizes the customer to send requests to protected resources. Get it by [authenticating as a customer](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-customers/authenticating-as-a-customer.html).  |
+
+Request sample: `DELETE https://glue.mysprykershop.com/wishlists/09264b7f-1894-58ed-81f4-d52d683e910a/wishlist-items/064_18404924`
+
+### Response
+
+If the item is removed successfully, the endpoint returns the `204 No Content` status code.
+
+## Possible errors
+
+| CODE | REASON |
+| --- | --- |
+| 201 | Cannot find the wishlist. |
+| 206 | Cannot add an item to the wishlist. |
+| 207 | Cannot remove the item. |
+| 208 | An item with the provided SKU does not exist in the wishlist. |
+
+To view generic errors that originate from the Glue Application, see [Reference information: GlueApplication errors](/docs/scos/dev/glue-api-guides/{{page.version}}/reference-information-glueapplication-errors.html).
```

Comment:
Also we can update wishlist item by `path: "/wishlists/{wishlistId}/wishlist-items/{wishlistItemId}"`
```
"parameters": [{
                  "ref": "acceptLanguage"
              }],
"responses": {
                  "403": "Unauthorized request.",
                  "404": "Wishlist not found.",
                  "422": "Cannot update the wishlist list item."
 },
```